### ISSUE TITLE
regression tests failing (dials.symmetry)

### ISSUE THREAD

**Issue by Anthchirp** [xia2/xia2#346]
https://jenkins.diamond.ac.uk/view/DIALS/job/xia2_regression/lastCompletedBuild/label=vm-rhel6w-1/testReport/Test.regression/test_small_molecule/test_dials/

```
xia2 pipeline=dials nproc=2 small_molecule=True read_all_image_headers=False trust_beam_centre=True /dls/science/groups/scisoft/DIALS/dials_data/small_molecule_example
```
```
-------------------- Preparing DEFAULT ---------------------
Error: dials.symmetry terminated with an error: see /scratch/jenkins_agent/workspace/xia2_regression/label/vm-rhel6w-1/tests/pytest/popen-gw3/test_dials0/DEFAULT/scale/14_dials.symmetry.log for more details
Please send the contents of xia2.txt, xia2.error and xia2-debug.txt to:
xia2.support@gmail.com
Traceback (most recent call last):
  File "/scratch/jenkins_agent/workspace/xia2_regression/label/vm-rhel6w-1/build_dials/build/../modules/xia2/command_line/xia2_main.py", line 365, in run
    xia2_main()
  File "/scratch/jenkins_agent/workspace/xia2_regression/label/vm-rhel6w-1/build_dials/build/../modules/xia2/command_line/xia2_main.py", line 272, in xia2_main
    Chatter.write(xinfo.get_output(), strip=False)
  File "/scratch/jenkins_agent/workspace/xia2_regression/label/vm-rhel6w-1/build_dials/modules/xia2/Schema/XProject.py", line 131, in get_output
    result += self._crystals[crystal].get_output()
  File "/scratch/jenkins_agent/workspace/xia2_regression/label/vm-rhel6w-1/build_dials/modules/xia2/Schema/XCrystal.py", line 507, in get_output
    reflections_all = self.get_scaled_merged_reflections()
  File "/scratch/jenkins_agent/workspace/xia2_regression/label/vm-rhel6w-1/build_dials/modules/xia2/Schema/XCrystal.py", line 952, in get_scaled_merged_reflections
    return self._get_scaler().get_scaled_merged_reflections()
  File "/scratch/jenkins_agent/workspace/xia2_regression/label/vm-rhel6w-1/build_dials/modules/xia2/Schema/Interfaces/Scaler.py", line 668, in get_scaled_merged_reflections
    self.scale()
  File "/scratch/jenkins_agent/workspace/xia2_regression/label/vm-rhel6w-1/build_dials/modules/xia2/Schema/Interfaces/Scaler.py", line 639, in scale
    self._scale_prepare()
  File "/scratch/jenkins_agent/workspace/xia2_regression/label/vm-rhel6w-1/build_dials/modules/xia2/Modules/Scaler/DialsScaler.py", line 336, in _scale_prepare
    need_to_return = self._standard_scale_prepare()
  File "/scratch/jenkins_agent/workspace/xia2_regression/label/vm-rhel6w-1/build_dials/modules/xia2/Modules/Scaler/DialsScaler.py", line 190, in _standard_scale_prepare
    [experiment], [reflections], [refiner]
  File "/scratch/jenkins_agent/workspace/xia2_regression/label/vm-rhel6w-1/build_dials/modules/xia2/Modules/Scaler/DialsScaler.py", line 923, in _dials_symmetry_indexer_jiffy
    experiments, reflections, refiners, multisweep
  File "/scratch/jenkins_agent/workspace/xia2_regression/label/vm-rhel6w-1/build_dials/modules/xia2/Modules/Scaler/DialsScaler.py", line 1079, in dials_symmetry_indexer_jiffy
    experiments, reflections
  File "/scratch/jenkins_agent/workspace/xia2_regression/label/vm-rhel6w-1/build_dials/modules/xia2/Modules/Scaler/DialsScaler.py", line 1148, in dials_symmetry_decide_pointgroup
    symmetry_analyser.decide_pointgroup()
  File "/scratch/jenkins_agent/workspace/xia2_regression/label/vm-rhel6w-1/build_dials/modules/xia2/Wrappers/Dials/Symmetry.py", line 227, in decide_pointgroup
    self.check_for_errors()
  File "/scratch/jenkins_agent/workspace/xia2_regression/label/vm-rhel6w-1/build_dials/modules/xia2/Driver/DefaultDriver.py", line 327, in check_for_errors
    self.check_for_error_text(self._standard_output_records[-30:])
  File "/scratch/jenkins_agent/workspace/xia2_regression/label/vm-rhel6w-1/build_dials/modules/xia2/Driver/DefaultDriver.py", line 315, in check_for_error_text
    % (os.path.basename(self._executable), self.get_log_file())
RuntimeError: dials.symmetry terminated with an error: see /scratch/jenkins_agent/workspace/xia2_regression/label/vm-rhel6w-1/tests/pytest/popen-gw3/test_dials0/DEFAULT/scale/14_dials.symmetry.log for more details
```

**Comment by Anthchirp** [comment]
likely culprit 14d72cc96d9e3548cb7a170f0a257561a923b5d6

**Comment by rjgildea** [comment]
This failure was actually triggered by https://github.com/dials/dials/commit/a16543e9e5db4ea226c7cbe47f7414879fd4866b, although I don't think the commit itself can be to blame, just likely resulting different output from `dials.integrate` triggers some pre-existing bug in `dials.symmetry` systematic absences code. One for @jbeilstenedmands I suspect.

**Comment by rjgildea** [comment]
Relevant error in dials.symmetry.log:
```
Analysing systematic absences

Laue group: P 1 2/m 1
Performing systematic absence checks on unscaled profile-integrated data
Read 38162 predicted reflections
Selected 29560 profile integrated reflections
Removed 566 reflections with d < 0.62
Combined 726 partial reflections with other partial reflections
Removed 321 reflections below partiality threshold
Laue group: P 1 2/m 1
Traceback (most recent call last):
  File "/Users/rjgildea/software/cctbx/build/../modules/dials/command_line/symmetry.py", line 347, in <module>
    run()
  File "/Users/rjgildea/software/cctbx/build/../modules/dials/command_line/symmetry.py", line 340, in run
    symmetry(experiments, reflections, params=params)
  File "/Users/rjgildea/software/cctbx/build/../modules/dials/command_line/symmetry.py", line 225, in symmetry
    float(params.systematic_absences.significance_level),
  File "/Users/rjgildea/software/cctbx/modules/dials/algorithms/symmetry/absences/run_absences_checks.py", line 35, in run_systematic_absences_checks
    laue_groups[laue_group], merged_reflections, significance_level
  File "/Users/rjgildea/software/cctbx/modules/dials/algorithms/symmetry/absences/laue_groups_info.py", line 32, in score_screw_axes
    scores.append(a.score_axis(reflection_table, significance_level))
  File "/Users/rjgildea/software/cctbx/modules/dials/algorithms/symmetry/absences/screw_axes.py", line 152, in score_axis
    mean_i_sigma_abs = flex.mean(sorted_exp_abs[:-1])
RuntimeError: Please report this error to dials-support@lists.sourceforge.net: mean() argument is an empty array
# command line:
# dials.symmetry  ''/Users/rjgildea/tmp/346b/DEFAULT/NATIVE/SWEEP1/integrate/12_integrated.expt'' ''/Users/rjgildea/tmp/346b/DEFAULT/NATIVE/SWEEP1/integrate/12_integrated.refl'' 'output.experiments='14_symmetrized.expt'' 'output.reflections='14_symmetrized.refl'' 'relative_length_tolerance=0.05' 'absolute_angle_tolerance=2' 'output.json='14_dials_symmetry.json''
```

**Comment by jbeilstenedmands** [comment]
Should be fixed with dials/dials@4dd49a7
Test passed locally, will wait to see if Jenkins agrees before closing.

**Comment by Anthchirp** [comment]
Fixed. Thank you
